Doc/howto/clinic.rst

```diff
@@ -582,7 +582,7 @@ How-to guides
 =============
 
 
-How to to rename C functions and variables generated by Argument Clinic
+How to rename C functions and variables generated by Argument Clinic
```

Comment:
Can you also update the line below the heading?

Reply:
Sorry, my bad.

Reply:
Thanks :)